### PR TITLE
Support s01e03 Follow Up

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

**Review by as2388 — CHANGES_REQUESTED** [review]
KeyError: "Code scheme 'KAKUMA WS - Correct Dataset' (id 'Scheme-fffffc6e') does not contain a code with match value 'leap kalobeyei girls empowerment'"

File: configurations/coding_plans.py [hunk 1]
```diff
@@ -323,6 +323,23 @@ def get_rqa_coding_plans(pipeline_name):
                ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET.get_code_with_match_value("leap kalobeyei currently in school"),
                raw_field_fold_strategy=FoldStrategies.assert_equal),
 
+    CodingPlan(raw_field="kalobeyei_girls_empowerment_raw",
+               dataset_name="kalobeyei_girls_empowerment",
+               time_field="kalobeyei_girls_empowerment_time",
+               coda_filename="WUSC-LEAP_girls_empowerment.json",
+               coding_configurations=[
+                   CodingConfiguration(
+                       coding_mode=CodingModes.SINGLE,
```
[the remark this answers is not in the document]
Comment (DanielMwendwa) [reply configurations/coding_plans.py]: Thanks, confirmed to be "multiple" and updated

File: configurations/coding_plans.py [hunk 2]
```diff
@@ -323,6 +323,23 @@ def get_rqa_coding_plans(pipeline_name):
...
+                       code_scheme=CodeSchemes.KALOBEYEI_GIRLS_EMPOWERMENT,
+                       coded_field="kalobeyei_girls_empowerment_coded",
+                       analysis_file_key="kalobeyei_girls_empowerment",
+                       fold_strategy=FoldStrategies.assert_label_ids_equal,
```
[the remark this answers is not in the document]
Comment (DanielMwendwa) [reply configurations/coding_plans.py]: Thanks, updated

**Comment by DanielMwendwa** [comment]
> fffffc6e

Resolved

**Comment by DanielMwendwa** [comment]
> KeyError: "Code scheme 'KAKUMA WS - Correct Dataset' (id 'Scheme-fffffc6e') does not contain a code with match value 'leap kalobeyei girls empowerment'"

Resolved

**Review by as2388 — APPROVED** [review]

**Pull request merged**